charts/opentelemetry-collector/templates/_custom_helpers.tpl

```diff
@@ -90,8 +90,17 @@ Determine the command to use based on platform and configuration.
 {{- if $configArg }}
 - {{ $configArg }}
 {{- end }}
+{{- $featureGates := list }}
 {{- if (and (.Values.presets.profilesCollection.enabled) (not .Values.presets.fleetManagement.enabled)) }}
-- "--feature-gates=+service.profilesSupport"
+{{- $featureGates = append $featureGates "+service.profilesSupport" }}
+{{- end }}
+{{- if or .Values.presets.spanMetrics.enabled .Values.presets.spanMetricsMulti.enabled }}
+{{- $featureGates = append $featureGates "+connector.spanmetrics.useSecondAsDefaultMetricsUnit" }}
+{{- $featureGates = append $featureGates "+connector.spanmetrics.excludeResourceMetrics" }}
+{{- $featureGates = append $featureGates "+spanmetrics.statusCodeConvention.useOtelPrefix" }}
+{{- end }}
+{{- if $featureGates }}
+- "--feature-gates={{ $featureGates | join "," }}"
```

Comment:
Why do we enable these feature flags?

i dont think we need to. Lets just prepare once it gets rolled out by default